tutorials/classification_pathmnist_notebook/config.yaml

```diff
@@ -0,0 +1,200 @@
+# affix version
+version:
+  {
+    minimum: 0.0.14,
+    maximum: 0.0.14 # this should NOT be made a variable, but should be tested after every tag is created

```

Comment:
The original tutorial that I had based this off of was in Google Colab and used the 0.0.14 version since the most recent version wasn't supported in Colab. I forgot to change the config.yaml file when I changed the tutorial to use the 0.0.16 version, but I will get to that now.